jekyll/_cci2/optimization-cookbook.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: classic-docs
+title: "CircleCI Optimizations Cookbook"
+short-title: "Optimizations Cookbook"
+description: "Starting point for Optimizations Cookbook"
+categories: [getting-started]
+order: 1
+---
+
+The *CircleCI Optimizations Cookbook* is a collection of individual use cases (referred to as "recipes") that provide you with detailed, step-by-step instructions on how to perform various optimization tasks using CircleCI resources. This guide, and it associated sections, will enable you to quickly and easily perform easily repeatable optimization tasks on the CircleCI platform.
```

Comment:
Who is this doc for, and why should I read it?

What are the concrete use-cases I will get out of this document. An example of this would be, reducing build minutes or developer time. And how.

My usual thought pattern when looking through examples or cookbooks, recipes, etc; is asking "What problem does this solve? And is that a problem I have now?"

An example of this might be; "OOoo, a recipe for baked potatoes; hmmm I have a problem in that we bought too many potatoes! Perfect."

Reply:
Rewrote introduction to clarify the purpose of this guide.

Reply:
Fixed in document.